commands/diff_test.go

```diff
@@ -4,6 +4,8 @@ import (
 	"io"
 	"testing"
 
+	"github.com/10gen/realm-cli/utils/telemetry"
+
```

Comment:
same thing here, can you fixup the import groups (not sure why/how we missed this in the first go-round)

Reply:
looks like there's still an extra newline here on L#8 (`"github.com/10gen/realm-cli/utils/telemetry"` can be grouped with the other "project imports" in the middle group here)